ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.html

```diff
@@ -89,6 +89,13 @@ <h5>No results</h5>
         "
       ></ess-sort-by-functionality>
     </div>
+   <ng-container  *ngIf="this.type === 'training'">
```

Comment:
Why this condition is based on the type of the first result? 
If all_collection would have a training item as the first result the banner should appear?
We can get the collection from the `CustomRoute`

Reply:
this box with url is only for the training collection. However, I based it on CustomRoute and collections after your comment